tox.ini

```diff
@@ -17,19 +18,19 @@ ignore_basepython_conflict = true
 deps = {[base]testdeps}
 commands = pytest --ignore=tests/test_plotly_extra.py tests
 
-[testenv:py36-plotly]
+[testenv:py38-plotly]
 extras = plotly
 commands = pytest tests/test_plotly_extra.py
 
-[testenv:py36-lint]
+[testenv:py37-lint]
```

Comment:
Not really. As far as I know, the lint should run with the oldest version to ensure we do not use features that are not available in this version but this issue is quite unlikely anyway (and if it's the case the tests will probably fail with a SyntaxError anyway).

Reply:
If @mgeplf is ok with it I would suggest converting this to py38 for homogeneity.

Reply:
I'm mostly indifferent, whatever works best for you guys.  I like homogeneity though, and since it's being changed, may as well bump what other projects are using.

Reply:
Ok, I am going to set py38 for homogeneity then.